test/performance/specs/site-editor.spec.js

```diff
@@ -185,22 +192,31 @@ test.describe( 'Site Editor Performance', () => {
 			categories: [ 'devtools.timeline' ],
 		} );
 
-		// Type "x" 200 times.
-		const typingSequence = new Array( 200 ).fill( 'x' ).join( '' );
-		await page.keyboard.type( typingSequence );
+		// The first character typed triggers a longer time (isTyping change).
+		// It can impact the stability of the metric, so we exclude it. It
+		// probably deserves a dedicated metric itself, though.
+		const samples = 10;
+		const throwaway = 1;
+		const rounds = samples + throwaway;
+
+		// Type the testing sequence into the empty paragraph.
+		await page.keyboard.type( 'x'.repeat( rounds ), {
+			delay: BROWSER_IDLE_WAIT,
+		} );
 
 		// Stop tracing and save results.
 		await browser.stopTracing();
```

Comment:
The tracing API also returns the [buffer](https://playwright.dev/docs/api/class-browser#browser-stop-tracing). So maybe we don't need to store this one-off trace to a file on disk after all and can just process them in memory.

Reply:
I've noticed that the trace file is not actually attached to the failure artifacts (in case the test fails mid-tracing), so it should be OK to only use the buffer instead.